onboarding/array.py

```diff
@@ -9,7 +9,14 @@ def find_min_index(arr):
     '''
 
     # insert code here
-    pass
+    min = arr[0]
```

Comment:
What happens if array is empty?

Reply:
Since there was nothing given, i considered that the array is non-empty. If it is not then could u suggest what I should return? Will None be fine?

Reply:
That works 👍